src/Http/Http.Abstractions/src/Validation/ValidateContext.cs

```diff
@@ -60,11 +62,18 @@ public sealed class ValidateContext
     /// </summary>
     public int CurrentDepth { get; set; }
 
-    internal void AddValidationError(string key, string[] error)
+    /// <summary>
+    /// Gets or sets the JSON serializer options to use for property name formatting.
+    /// When available, property names in validation errors will be formatted according to the
+    /// PropertyNamingPolicy and JsonPropertyName attributes.
+    /// </summary>
+    public JsonSerializerOptions? SerializerOptions { get; set; }
```

Comment:
Putting it on the record that I'm not a fan of adding this to the `ValidateContext`. Especially if we're wanting to eventually move this API into dotnet/runtime.